docs/howto/windows.md

```diff
@@ -106,28 +87,132 @@ many of these.)
   but it may be a bit simpler to use the Linux version.
 * One significant limitation of WSL (as of 2018) is that doing a large
   amount of filesystem I/O tends to be slow.  This is a reason to do
-  operations like a Gradle build, perhaps `git` commands, and perhaps
+  operations like a Gradle build ( Gradle is the Java program that does the Android build), perhaps `git` commands, and perhaps
   the occasional `yarn install` using the Windows versions.
 
 [so-guide]: https://stackoverflow.com/questions/42614347/running-react-native-in-wsl-with-the-emulator-running-directly-in-windows
 
-### Accessing the filesystem
-You can access your Windows local drives from `bash` and other Linux
-programs by navigating to `/mnt/<drive-letter>` .
 
-### Running `yarn` commands
-First, install Node.js and Yarn in your Linux environment.  You'll follow
-the same steps as in [our main setup guide](build-run.md) -- just use the
-Linux versions. :-)
+### Step-by-step instructions for setting up WSL
+
+1. Open Powershell as an Administrator and run the command below to enable WSL. 
+
+    `Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Windows-Subsystem-Linux`
+
+2. Install [Chocolatey][chocolatey]. After installing Chocolatey, use it to install the JAVA SE Development Kit with  
+
+    `choco install -y jdk8`
```

Comment:
Huh, also I am curious why you needed this step at all. I'd expect that the Android Studio installer would install a suitable JDK if you don't already have one.